src/components/Tasks/Toolbar.tsx

```diff
@@ -35,7 +50,19 @@ export default function Toolbar(props: PropsType) {
 
   return (
     <div style={{ display: 'flex', justifyContent: 'space-between' }}>
-      {defaultCollection && <QuickAdd style={{ flexGrow: 1, marginRight: '0.75em' }} onSubmit={onItemSave} defaultCollection={defaultCollection} />}
+      {defaultCollection && <QuickAdd style={{ flexGrow: 1, marginRight: '1.75em' }} onSubmit={onItemSave} defaultCollection={defaultCollection} />}
+
+      <FormControl style={{ marginRight: '0.75em' }}>
+        <InputLabel>Sort by</InputLabel>
```

Comment:
This looks terrible. It shouldn't be an `InputLabel`... Just make it a sort icon that opens the menu...
After you do that you will probably want to change the top margin right accordingly.
And change the menu item that is currently selected to indicate it in some way.